Packs/FortinetFortiwebVM/Integrations/FortinetFortiwebVM/FortinetFortiwebVM.yml

```diff
@@ -0,0 +1,2407 @@
+    - description: Geo IP Exception groups.
+      isArray: false
+      name: exception_rule
+      required: false
+    - auto: PREDEFINED
+      description: "Select which action FortiWeb will take when it detects a violation of the rule:alert_deny—Block the request (or reset the connection) and generate an alert and/or log message.deny_no_log—Block the request (or reset the connection).block-period—Block subsequent requests from the client for a number of seconds. Also configure block-period. Supports API version 2 only."
+      isArray: false
+      name: action
+      predefined:
+      - Alert deny
+      - Block period
+      - Deny (no log)
+      required: false
+    - description: "Block Period.The number of seconds that you want to block subsequent requests.The valid range is 1–3,600 seconds. Supports API version 2 only."
+      isArray: false
+      name: block_period
+      required: false
+    - auto: PREDEFINED
+      predefined:
+      - enable
+      - disable
+      description: "enable Ignore X-Forwarded-For so that the IP addresses will be scanned at the TCP layer instead of HTTP layer. Supports API version 2 only."
+      isArray: false
+      name: ignore_x_forwarded_for
+      required: false
+    deprecated: false
+    description: Update Geo IP.
+    execution: false
+    name: fortiwebvm-geo-ip-group-update
+  - arguments:
+    - description: Geo IP Group Name.
+      isArray: false
+      name: name
+      required: true
+    deprecated: false
+    description: Delete Geo IP.
+    execution: false
+    name: fortiwebvm-geo-ip-group-delete
+  - arguments:
+    - description: Geo IP Group Name.
+      isArray: false
+      name: name
+      required: false
+    - description: The page number of the results to retrieve.
+      isArray: false
+      name: page
+      required: false
+    - description: The maximum number of records to retrieve per page.
+      isArray: false
+      name: page_size
+      required: false
+    - defaultValue: '50'
+      description: The maximum number of records to retrieve.
+      isArray: false
+      name: limit
+      required: false
+    deprecated: false
+    description: Get Geo IP list.
+    execution: false
+    name: fortiwebvm-geo-ip-group-list
+    outputs:
+    - contextPath: FortiwebVM.GeoIpGroup.id
+      description: Geo IP Group ID.
+      type: Number
+    - contextPath: FortiwebVM.GeoIpGroup.count
+      description: Geo IP Group Members Count.
+      type: Number
+    - contextPath: FortiwebVM.GeoIpGroup.trigger_policy
+      description: Geo IP Group Trigger Policy Name.
+      type: String
+    - contextPath: FortiwebVM.GeoIpGroup.severity
+      description: Geo IP Group Severity Number.
+      type: String
+    - contextPath: FortiwebVM.GeoIpGroup.except
+      description: 'Geo IP Group Exception groups.'
+      type: String
+    - contextPath: FortiwebVM.GeoIpGroup.can_delete
+      description: Geo IP Group Can delete flag. Supports API version 1 only.
```

Comment:
Don't understand this description.  Should it be: "Whether the Geo IP group can be deleted. "?

Reply:
Yes.